### ISSUE TITLE
Redundant Object

### ISSUE THREAD

**Issue by volodya-lombrozo** [objectionary/lints#618]
I have the following xmir:

<details>
<summary>MethodByte.xmir</summary>
```xmll
<?xml version="1.0" encoding="UTF-8"?>
<object xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        dob="2023-03-19T00:00:00"
        ms="0"
        revision="1234567"
        time="2025-05-26T12:02:06.330999Z"
        version="1.2.3"
        xsi:noNamespaceSchemaLocation="file:////Users/lombrozo/Workspace/EOlang/jeo-maven-plugin/target/xsd/XMIR-0.56.2.xsd">
   <listing>// class version 55.0 (55)
// access flags 0x21
public class org/eolang/jeo/MethodByte {

  // compiled from: MethodByte.java

  // access flags 0x1
  public &lt;init&gt;()V
   L0
    LINENUMBER 3 L0
    ALOAD 0
    INVOKESPECIAL java/lang/Object.&lt;init&gt; ()V
    RETURN
    MAXSTACK = 1
    MAXLOCALS = 1

  // access flags 0x9
  public static main([Ljava/lang/String;)V
   L0
    LINENUMBER 5 L0
    GETSTATIC java/lang/System.out : Ljava/io/PrintStream;
    INVOKESTATIC org/eolang/jeo/MethodByte.method ()B
    INVOKEVIRTUAL java/io/PrintStream.println (I)V
   L1
    LINENUMBER 6 L1
    GETSTATIC java/lang/System.out : Ljava/io/PrintStream;
    LDC "passed"
    INVOKEVIRTUAL java/io/PrintStream.println (Ljava/lang/String;)V
   L2
    LINENUMBER 7 L2
    RETURN
    MAXSTACK = 2
    MAXLOCALS = 1

  // access flags 0x8
  static method()B
   L0
    LINENUMBER 9 L0
    BIPUSH 52
    IRETURN
    MAXSTACK = 1
    MAXLOCALS = 0
}
</listing>
   <metas>
      <meta>
         <head>home</head>
         <tail>https://github.com/objectionary/jeo-maven-plugin</tail>
         <part>https://github.com/objectionary/jeo-maven-plugin</part>
      </meta>
      <meta>
         <head>package</head>
         <tail>j$org.j$eolang.j$jeo</tail>
         <part>j$org.j$eolang.j$jeo</part>
      </meta>
      <meta>
         <head>spdx</head>
         <tail>SPDX-License-Identifier: MIT</tail>
         <part>SPDX-License-Identifier:</part>
         <part>MIT</part>
      </meta>
      <meta>
         <head>version</head>
         <tail>1.2.3</tail>
      </meta>
   </metas>
   <o base="Q.jeo.class" name="j$MethodByte">
      <o as="version" base="Q.org.eolang.number">
         <o base="Q.org.eolang.bytes">
            <o>40-4B-80-00-00-00-00-00</o>
         </o>
      </o>
      <o as="access" base="Q.org.eolang.number">
         <o base="Q.org.eolang.bytes">
            <o>40-40-80-00-00-00-00-00</o>
         </o>
      </o>
      <o as="supername" base="Q.org.eolang.string"><!-- "java/lang/Object" -->
         <o base="Q.org.eolang.bytes">
            <o>6A-61-76-61-2F-6C-61-6E-67-2F-4F-62-6A-65-63-74</o>
         </o>
      </o>
      <o as="interfaces" base="Q.jeo.seq.of0"/>
      <o as="j$object@init@" base="Q.jeo.method">
         <o as="access" base="Q.org.eolang.number">
            <o base="Q.org.eolang.bytes">
               <o>3F-F0-00-00-00-00-00-00</o>
            </o>
         </o>
         <o as="descriptor" base="Q.org.eolang.string"><!-- "()V" -->
            <o base="Q.org.eolang.bytes">
               <o>28-29-56</o>
            </o>
         </o>
         <o as="signature" base="Q.org.eolang.string"><!-- "" -->
            <o base="Q.org.eolang.bytes">
               <o>--</o>
            </o>
         </o>
         <o as="maxs" base="Q.jeo.maxs">
            <o base="Q.org.eolang.number">
               <o base="Q.org.eolang.bytes">
                  <o>3F-F0-00-00-00-00-00-00</o>
               </o>
            </o>
            <o base="Q.org.eolang.number">
               <o base="Q.org.eolang.bytes">
                  <o>3F-F0-00-00-00-00-00-00</o>
               </o>
            </o>
         </o>
         <o as="body" base="Q.jeo.seq.of3">
            <o base="Q.jeo.opcode.aload"><!-- #25:aload(0) -->
               <o base="Q.org.eolang.number">
                  <o base="Q.org.eolang.bytes">
                     <o>40-39-00-00-00-00-00-00</o>
                  </o>
               </o>
               <o base="Q.org.eolang.number">
                  <o base="Q.org.eolang.bytes">
                     <o>00-00-00-00-00-00-00-00</o>
                  </o>
               </o>
            </o>
            <o base="Q.jeo.opcode.invokespecial"><!-- #183:invokespecial(java/lang/Object, &lt;init&gt;, ()V, false) -->
               <o base="Q.org.eolang.number">
                  <o base="Q.org.eolang.bytes">
                     <o>40-66-E0-00-00-00-00-00</o>
                  </o>
               </o>
               <o base="Q.org.eolang.string"><!-- "java/lang/Object" -->
                  <o base="Q.org.eolang.bytes">
                     <o>6A-61-76-61-2F-6C-61-6E-67-2F-4F-62-6A-65-63-74</o>
                  </o>
               </o>
               <o base="Q.org.eolang.string"><!-- "&lt;init&gt;" -->
                  <o base="Q.org.eolang.bytes">
                     <o>3C-69-6E-69-74-3E</o>
                  </o>
               </o>
               <o base="Q.org.eolang.string"><!-- "()V" -->
                  <o base="Q.org.eolang.bytes">
                     <o>28-29-56</o>
                  </o>
               </o>
               <o base="Q.org.eolang.false"/>
            </o>
            <o base="Q.jeo.opcode.return"><!-- #177:return() -->
               <o base="Q.org.eolang.number">
                  <o base="Q.org.eolang.bytes">
                     <o>40-66-20-00-00-00-00-00</o>
                  </o>
               </o>
            </o>
         </o>
      </o>
      <o as="j$main" base="Q.jeo.method">
         <o as="access" base="Q.org.eolang.number">
            <o base="Q.org.eolang.bytes">
               <o>40-22-00-00-00-00-00-00</o>
            </o>
         </o>
         <o as="descriptor" base="Q.org.eolang.string"><!-- "([Ljava/lang/String;)V" -->
            <o base="Q.org.eolang.bytes">
               <o>28-5B-4C-6A-61-76-61-2F-6C-61-6E-67-2F-53-74-72-69-6E-67-3B-29-56</o>
            </o>
         </o>
         <o as="signature" base="Q.org.eolang.string"><!-- "" -->
            <o base="Q.org.eolang.bytes">
               <o>--</o>
            </o>
         </o>
         <o as="maxs" base="Q.jeo.maxs">
            <o base="Q.org.eolang.number">
               <o base="Q.org.eolang.bytes">
                  <o>40-00-00-00-00-00-00-00</o>
               </o>
            </o>
            <o base="Q.org.eolang.number">
               <o base="Q.org.eolang.bytes">
                  <o>3F-F0-00-00-00-00-00-00</o>
               </o>
            </o>
         </o>
         <o as="params" base="Q.jeo.params">
            <o as="arg0" base="Q.jeo.param">
               <o as="index" base="Q.org.eolang.number">
                  <o base="Q.org.eolang.bytes">
                     <o>00-00-00-00-00-00-00-00</o>
                  </o>
               </o>
               <o as="access" base="Q.org.eolang.number">
                  <o base="Q.org.eolang.bytes">
                     <o>00-00-00-00-00-00-00-00</o>
                  </o>
               </o>
               <o as="type" base="Q.org.eolang.string"><!-- "[Ljava/lang/String;" -->
                  <o base="Q.org.eolang.bytes">
                     <o>5B-4C-6A-61-76-61-2F-6C-61-6E-67-2F-53-74-72-69-6E-67-3B</o>
                  </o>
               </o>
            </o>
         </o>
         <o as="body" base="Q.jeo.seq.of7">
            <o base="Q.jeo.opcode.getstatic"><!-- #178:getstatic(java/lang/System, out, Ljava/io/PrintStream;) -->
               <o base="Q.org.eolang.number">
                  <o base="Q.org.eolang.bytes">
                     <o>40-66-40-00-00-00-00-00</o>
                  </o>
               </o>
               <o base="Q.org.eolang.string"><!-- "java/lang/System" -->
                  <o base="Q.org.eolang.bytes">
                     <o>6A-61-76-61-2F-6C-61-6E-67-2F-53-79-73-74-65-6D</o>
                  </o>
               </o>
               <o base="Q.org.eolang.string"><!-- "out" -->
                  <o base="Q.org.eolang.bytes">
                     <o>6F-75-74</o>
                  </o>
               </o>
               <o base="Q.org.eolang.string"><!-- "Ljava/io/PrintStream;" -->
                  <o base="Q.org.eolang.bytes">
                     <o>4C-6A-61-76-61-2F-69-6F-2F-50-72-69-6E-74-53-74-72-65-61-6D-3B</o>
                  </o>
               </o>
            </o>
            <o base="Q.jeo.opcode.invokestatic"><!-- #184:invokestatic(org/eolang/jeo/MethodByte, method, ()B, false) -->
               <o base="Q.org.eolang.number">
                  <o base="Q.org.eolang.bytes">
                     <o>40-67-00-00-00-00-00-00</o>
                  </o>
               </o>
               <o base="Q.org.eolang.string"><!-- "org/eolang/jeo/MethodByte" -->
                  <o base="Q.org.eolang.bytes">
                     <o>6F-72-67-2F-65-6F-6C-61-6E-67-2F-6A-65-6F-2F-4D-65-74-68-6F-64-42-79-74-65</o>
                  </o>
               </o>
               <o base="Q.org.eolang.string"><!-- "method" -->
                  <o base="Q.org.eolang.bytes">
                     <o>6D-65-74-68-6F-64</o>
                  </o>
               </o>
               <o base="Q.org.eolang.string"><!-- "()B" -->
                  <o base="Q.org.eolang.bytes">
                     <o>28-29-42</o>
                  </o>
               </o>
               <o base="Q.org.eolang.false"/>
            </o>
            <o base="Q.jeo.opcode.invokevirtual"><!-- #182:invokevirtual(java/io/PrintStream, println, (I)V, false) -->
               <o base="Q.org.eolang.number">
                  <o base="Q.org.eolang.bytes">
                     <o>40-66-C0-00-00-00-00-00</o>
                  </o>
               </o>
               <o base="Q.org.eolang.string"><!-- "java/io/PrintStream" -->
                  <o base="Q.org.eolang.bytes">
                     <o>6A-61-76-61-2F-69-6F-2F-50-72-69-6E-74-53-74-72-65-61-6D</o>
                  </o>
               </o>
               <o base="Q.org.eolang.string"><!-- "println" -->
                  <o base="Q.org.eolang.bytes">
                     <o>70-72-69-6E-74-6C-6E</o>
                  </o>
               </o>
               <o base="Q.org.eolang.string"><!-- "(I)V" -->
                  <o base="Q.org.eolang.bytes">
                     <o>28-49-29-56</o>
                  </o>
               </o>
               <o base="Q.org.eolang.false"/>
            </o>
            <o base="Q.jeo.opcode.getstatic"><!-- #178:getstatic(java/lang/System, out, Ljava/io/PrintStream;) -->
               <o base="Q.org.eolang.number">
                  <o base="Q.org.eolang.bytes">
                     <o>40-66-40-00-00-00-00-00</o>
                  </o>
               </o>
               <o base="Q.org.eolang.string"><!-- "java/lang/System" -->
                  <o base="Q.org.eolang.bytes">
                     <o>6A-61-76-61-2F-6C-61-6E-67-2F-53-79-73-74-65-6D</o>
                  </o>
               </o>
               <o base="Q.org.eolang.string"><!-- "out" -->
                  <o base="Q.org.eolang.bytes">
                     <o>6F-75-74</o>
                  </o>
               </o>
               <o base="Q.org.eolang.string"><!-- "Ljava/io/PrintStream;" -->
                  <o base="Q.org.eolang.bytes">
                     <o>4C-6A-61-76-61-2F-69-6F-2F-50-72-69-6E-74-53-74-72-65-61-6D-3B</o>
                  </o>
               </o>
            </o>
            <o base="Q.jeo.opcode.ldc"><!-- #18:ldc(passed) -->
               <o base="Q.org.eolang.number">
                  <o base="Q.org.eolang.bytes">
                     <o>40-32-00-00-00-00-00-00</o>
                  </o>
               </o>
               <o base="Q.org.eolang.string"><!-- "passed" -->
                  <o base="Q.org.eolang.bytes">
                     <o>70-61-73-73-65-64</o>
                  </o>
               </o>
            </o>
            <o base="Q.jeo.opcode.invokevirtual"><!-- #182:invokevirtual(java/io/PrintStream, println, (Ljava/lang/String;)V, false) -->
               <o base="Q.org.eolang.number">
                  <o base="Q.org.eolang.bytes">
                     <o>40-66-C0-00-00-00-00-00</o>
                  </o>
               </o>
               <o base="Q.org.eolang.string"><!-- "java/io/PrintStream" -->
                  <o base="Q.org.eolang.bytes">
                     <o>6A-61-76-61-2F-69-6F-2F-50-72-69-6E-74-53-74-72-65-61-6D</o>
                  </o>
               </o>
               <o base="Q.org.eolang.string"><!-- "println" -->
                  <o base="Q.org.eolang.bytes">
                     <o>70-72-69-6E-74-6C-6E</o>
                  </o>
               </o>
               <o base="Q.org.eolang.string"><!-- "(Ljava/lang/String;)V" -->
                  <o base="Q.org.eolang.bytes">
                     <o>28-4C-6A-61-76-61-2F-6C-61-6E-67-2F-53-74-72-69-6E-67-3B-29-56</o>
                  </o>
               </o>
               <o base="Q.org.eolang.false"/>
            </o>
            <o base="Q.jeo.opcode.return"><!-- #177:return() -->
               <o base="Q.org.eolang.number">
                  <o base="Q.org.eolang.bytes">
                     <o>40-66-20-00-00-00-00-00</o>
                  </o>
               </o>
            </o>
         </o>
      </o>
      <o as="j$method" base="Q.jeo.method">
         <o as="access" base="Q.org.eolang.number">
            <o base="Q.org.eolang.bytes">
               <o>40-20-00-00-00-00-00-00</o>
            </o>
         </o>
         <o as="descriptor" base="Q.org.eolang.string"><!-- "()B" -->
            <o base="Q.org.eolang.bytes">
               <o>28-29-42</o>
            </o>
         </o>
         <o as="signature" base="Q.org.eolang.string"><!-- "" -->
            <o base="Q.org.eolang.bytes">
               <o>--</o>
            </o>
         </o>
         <o as="maxs" base="Q.jeo.maxs">
            <o base="Q.org.eolang.number">
               <o base="Q.org.eolang.bytes">
                  <o>3F-F0-00-00-00-00-00-00</o>
               </o>
            </o>
            <o base="Q.org.eolang.number">
               <o base="Q.org.eolang.bytes">
                  <o>00-00-00-00-00-00-00-00</o>
               </o>
            </o>
         </o>
         <o as="body" base="Q.jeo.seq.of2">
            <o base="Q.jeo.opcode.bipush"><!-- #16:bipush(52) -->
               <o base="Q.org.eolang.number">
                  <o base="Q.org.eolang.bytes">
                     <o>40-30-00-00-00-00-00-00</o>
                  </o>
               </o>
               <o base="Q.org.eolang.number">
                  <o base="Q.org.eolang.bytes">
                     <o>40-4A-00-00-00-00-00-00</o>
                  </o>
               </o>
            </o>
            <o base="Q.jeo.opcode.ireturn"><!-- #172:ireturn() -->
               <o base="Q.org.eolang.number">
                  <o base="Q.org.eolang.bytes">
                     <o>40-65-80-00-00-00-00-00</o>
                  </o>
               </o>
            </o>
         </o>
      </o>
   </o>
</object>
```
</details>

When I run:

```java
new Source(new StrictXmir(xmir)).defects()
```

I get the following error:

```
[j$org.j$eolang.j$jeo.j$MethodByte redundant-object (Single) WARNING] The object "j$MethodByte" is redundant, consider inline it instead],
```

What should I inline? I have only a top-level object with the name `j$MethodByte`/

**Comment by h1alexbel** [comment]
@volodya-lombrozo already aware of this problem here: #609
